manifests/package.pp

```diff
@@ -38,19 +38,22 @@
   if $elasticsearch::ensure == 'present' {
 
     # Create directory to place the package file
-    exec { 'create_package_dir_elasticsearch':
-      cwd     => '/',
-      path    => ['/usr/bin', '/bin'],
-      command => "mkdir -p ${elasticsearch::package_dir}",
-      creates => $elasticsearch::package_dir,
-    }
+    $package_dir = $elasticsearch::package_dir
```

Comment:
the package dir is used for puppet:// based plugins so its fairly mandatory to be created, otherwise it will fail.
See https://github.com/elastic/puppet-elasticsearch/blob/master/manifests/plugin.pp#L115